packages/ui/nuxt.config.js

```diff
@@ -153,9 +156,19 @@ export default {
       { path: '~/components/cards', extensions: ['vue', 'js'] },
     ],
   },
-
+  routes: [
+    {
+      src: '/sw.js',
```

Comment:
Is `sw.js` the default file created by `@nuxtjs/pwa`? We only need the cache and not all the other modules that `workbox` adds